internal/tools/redis/redis.go

```diff
@@ -176,7 +176,7 @@ func (t Tool) Authorized(verifiedAuthServices []string) bool {
 	return tools.IsAuthorized(t.AuthRequired, verifiedAuthServices)
 }
 
-// replaceCommandsParams is a helper function to replace parameters in the commands
+// Helper function to replace parameters in the commands
```

Comment:
```suggestion
// replaceCommandsParams  is a helper function to replace parameters in the commands
```

Golang's doc comment usually start with the name~